apps/files/src/components/FilesTopBar.vue

```diff
@@ -27,8 +27,11 @@
         </oc-menu>
 
         <oc-menu v-if="this.canUpload">
-          <oc-button id="new-file-menu-btn" text="+ New" slot="activator" variation="primary"></oc-button>
-          <!-- TODO: replace with oc-list elements-->
+          <template slot="activator">
```

Comment:
shorter version:
```
 <oc-button id="new-file-menu-btn" slot="activator" variation="primary"><translate>+ New</translate></oc-button>
```

If there is only on element which it to be used inside the slot the slot can be set as attribute directly

Reply:
Ok, right. I fixed it and also squashed the commits.